src/VirtualClient/VirtualClient.Actions.UnitTests/HPLinpack/HPLinpackMetricsParserTests.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace VirtualClient.Actions
+{
+    using VirtualClient.Common.Contracts;
+    using NUnit.Framework;
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Linq;
+    using System.Reflection;
+    using System.Text;
+    using System.Threading.Tasks;
+    using VirtualClient.Contracts;
+    using Microsoft.Identity.Client;
```

Comment:
Is the SchemaException coming from this namespace? Is there a normal schema exception you could use?

Reply:
It's from VirtualClient namespace.